code/game/world.dm

```diff
@@ -342,6 +342,8 @@ GLOBAL_VAR(restart_counter)
 		s += ": [jointext(features, ", ")]"
 
 	s += "<br>Round time: <b>[gameTimestamp("hh:mm")]</b>"
+	if(SSmapping.config)
+		s += "<br>Map: <b>[SSmapping.config.map_name]</b>"
```

Comment:
Could you implement a check to see if the map is in rotation, giving custom maps some kind of generic name? Admins historically have been pretty good about reasonable custom map names (as far as data lets me know, we can't see one-offs), but it's better to be safe than sorry with user-input strings that show up on the hub